src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -98,9 +100,19 @@ import Chainweb.Pact.Templates
 import Chainweb.Pact.Transactions.UpgradeTransactions (upgradeTransactions)
 import Chainweb.Pact.Types
 import Chainweb.Transaction
-import Chainweb.Utils (encodeToByteString, sshow)
+import Chainweb.Utils (encodeToByteString, sshow, tryAllSynchronous)
 import Chainweb.Version as V
 
+-- -------------------------------------------------------------------------- --
+-- NFData orphans for Pact types
+--
+-- TODO: these should be moved to Pact!
+
+deriving instance NFData PactResult
+deriving instance NFData RequestKey
+deriving instance NFData a => NFData (CommandResult a)
+
+-- -------------------------------------------------------------------------- --
```

Comment:
@emilypi could we include these in pact?

Reply:
Yeah, i'll open a PR with it today.